lab/src/Microsoft.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace Microsoft.ServiceModel.Syndication.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Xunit;
+    using Microsoft.ServiceModel.Syndication;
+    using System.Xml;
+    using System.IO;
+
+    public static class BasicScenarioTests
+    {
+                
+        [Fact]
+        public static void SyndicationFeed_CreateNewFeed()
+        {
+            // *** SETUP *** \\
+            SyndicationFeed sf = new SyndicationFeed("First feed on .net core ever!!", "This is the first feed on .net core ever!", new Uri("https://github.com/dotnet/wcf"));
+
+            XmlWriter xmlw = XmlWriter.Create("FirstFeedEver.xml");
+            Rss20FeedFormatter rssf = new Rss20FeedFormatter(sf);
+
+            // *** EXECUTE *** \\
+            rssf.WriteTo(xmlw);
+            xmlw.Close();
+
+            // *** VALIDATE *** \\
+            Assert.True(File.Exists("FirstFeedEver.xml"));
+
+            // *** CLEANUP *** \\
+            File.Delete("FirstFeedEver.xml");
```

Comment:
Please use a variable for "FirstFeedEver.xml", so you don't have to hard code it 3 times :). Same for other tests.

You can consider to use the test case name as the xml file name. The benefit is if anything went wrong, you will know which xml file to check.